index.html

```diff
@@ -809,185 +809,197 @@ <h3 class="modal-title" id="warningTitle">
-                  </div>
+          <div class="modal-body ps-4 pe-5">
+            <!-- TODO: getting rid of forms if they're not necessary? more reading required -->
+            <form>
+              <input id="talkId" type="hidden" value=""/>
+
+              <!-- TODO: propose we use either a placeholder OR help text but not both (when possible). see https://ux.stackexchange.com/questions/78459/should-help-text-be-above-or-below-the-input
+
+              this is apparently a more lengthy topic than I expected so this is rough notes. let's use placeholder text to suggest 
+              examples or expected format rather than just reiterating instructions (see 
+              https://uxplanet.org/the-anatomy-of-input-field-c3ef863e01d7)'
+              
+              -->
+              <div class="form-group mb-3">
+                <label for="newTalkTitle">
+                  Talk Title
+                </label>
+                <input type="text" name="newTalkTitle" placeholder="Talk title (required)" class="form-control required my-1" id="newTalkTitle" required />
+                <small class="form-text">
+                  This should be the title of the original paper.
+                </small>
+              </div>
+              <div id="talkTitleWarning">
+                <div class="alert alert-danger">Talk title is required.</div>
+              </div>
+
+              <div class="form-group mb-3">
+                <label for="newTalkAuthors">
+                  Talk Author(s)
+                </label>
+                <!-- TODO: shouldn't talk authors be a required field?' -->
+                <input type="text" name="newTalkAuthor" placeholder="Author names separated by and" class="form-control my-1"
+                  aria-describedby="newAuthorHelp" id="newTalkAuthor" />
+                <small id="newAuthorHelp" class="form-text">
+                  Use BibTeX formatting (i.e. names are separated by and).
+                </small>
+              </div>
+
+              <div class="form-group mb-3">
+                <label for="newTalkAffiliation">
+                  Affiliation
+                </label>
+                <input type="text" name="newTalkAffiliation" placeholder="Author affiliation(s)" class="form-control my-1"
+                  aria-describedby="newAffiliationHelp" id="newTalkAffiliation" />
+                <small id="newAffiliationHelp" class="form-text">
+                  Enter affiliations for authors, separated by semicolon. If authors share an affiliation, there is no need to enter
+                  the same institution twice.
+                </small>
+              </div>
+
+              <!-- TODO: this spacing on the div below will not work for this case -->
+              <div class="mb-3">
+                <label for="abstract">
+                  Abstract
+                </label>
+                <!-- TODO: I advocate for removing the button and just showing the frickin textarea -->
```

Comment:
I agree about omitting the button for abstract, now that it's scrolling. The textarea is sufficient. I might make it only 5 lines high instead of 8 (see how it looks). I would definitely make it full width in the dialog.